Options/WeakAurasOptions.lua

```diff
@@ -7126,13 +7126,21 @@ function WeakAuras.CreateFrame()
         if not(combinedText == "") then
           combinedText = combinedText.."\n\n";
         end
-        combinedText = combinedText.."--"..childId.."\n";
+
+        local formatString
```

Comment:
It's intentionally... I need the first and second part of the string later to split it up, and I need thos string fragments to be the same between here and there. This makes it so that the translators can't break that.